README.md

```diff
@@ -215,6 +215,40 @@ app.get('/.well-known/acme-challenge/:acmeToken', function(req, res, next) {
 });
 ```
 
+### PHP Apps
+
+Add the following to `.well-known/acme-challenge/index.php`
+
+```php
+<?php
+$request = $_SERVER['REQUEST_URI'];
+if(preg_match('#^/.well-known/acme-challenge/#', $request) === 0) {
+    return;
+}
+
+if(isset($_ENV['ACME_KEY'])) die(trim($_ENV['ACME_KEY']));
```

Comment:
The ruby example sets an array with key/value and loops through all entries, so handling one or multiple keys ends up being the same.
In the end, it checks for the token value against the URL though.

```ruby
data.each do |e|
  if env["PATH_INFO"] == "/.well-known/acme-challenge/#{e[:token]}"
    return [200, { "Content-Type" => "text/plain" }, [e[:key]]]
  end
end
```